### ISSUE TITLE
TypeAlias from typing_extensions is not recognised

### ISSUE THREAD

**Issue by layday** [microsoft/pyright#1161]
**Describe the bug**
The `TypeAlias` backport from `typing_extensions` cannot be used as a substitute for the Python 3.10 `TypeAlias`.

**To Reproduce**
Type check the snippet below.

**Expected behavior**
That `foo: TypeAlias = str` would not produce any errors.

**Screenshots or Code**
```python
from typing_extensions import TypeAlias

# Expression of type "Type[str]" cannot be assigned to declared type "TypeAlias"
#   Type "Type[str]" cannot be assigned to type "TypeAlias"
foo: TypeAlias = str
```

**VS Code extension or command-line**
Either, 1.1.85

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version.

(I appreciate you sending me some bugs with easy fixes. I've been working on some of your earlier bug reports off and on for over three weeks!)

**Comment by layday** [comment]
I'm happy to contribute in this small way - I've just sent another easily-fixable bug your way ;)

**Comment by erictraut** [comment]
This is now addressed in 1.1.86, which I just published. It will also be included in the next release of pylance.
